### ISSUE TITLE
intellij platform 2022 support missing

### ISSUE THREAD

**Issue by benjaminlewandowski** [kiteco/issue-tracker#829]

**Comment by PavelA85** [comment]
tried to add support but with no luck :(
https://github.com/kiteco/intellij-plugin/pull/2

**Comment by KnotFalse** [comment]
This issue is occurring with all JetBrains 2022 products.

The error is screenshotted below:

<img width="267" alt="image" src="https://user-images.githubusercontent.com/33779537/165824154-595ac34e-a3c6-462b-85ad-d1dde0538643.png">

This error was specifically generated in JetBrains GoLand, but I have had it occur in PHPStorm and PyCharm. My best guess as to why this error is occurring is because the way Kite checks for version compatibility is by regex or substring and it isn't expecting the prepended value before before the version number.

**Comment by bahyqn** [comment]
I found this article on the Internet. I can use the kite normally in the future according to method 1. https://blog.csdn.net/qq_43554335/article/details/121726226

**Comment by arnonmoscona** [comment]
Yep. It's been a while. No new plugin version yet. That's not great for a paid product

**Comment by kospl** [comment]
https://plugins.jetbrains.com/plugin/15148-kite-ai-code-autocomplete-python-java-js-go-html-php-c---more

> Not compatible with the version of your running IDE (PhpStorm 2022.1.2)

Is this product is still alive? Twitter is muted for a year already.

**Comment by arnonmoscona** [comment]
It looks pretty dead to me. At least based on the insights tab on this repo... does not look like any active development aside from sporadic PRs here and there. Also for about a year...

**Comment by arnonmoscona** [comment]
Last merged PR over 2 years ago
